docs/ref/models/indexes.txt

```diff
@@ -27,6 +27,10 @@ options`_.
 
     Creates an index (B-Tree) in the database.
 
+    .. attribute:: max_name_length
```

Comment:
I'm not sure if there's a need to document this. Most users shouldn't care about it unless they are writing their own indexes.